tests/python_tests/test_llm_pipeline_static.py

```diff
@@ -29,25 +51,66 @@ def generate_chat_history(model_path, device, pipeline_config, questions):
     return chat_history
 
 
+generation_configs = [
+    get_greedy(),
+    get_greedy_with_min_and_max_tokens(),
+    get_greedy_with_repetition_penalty(),
+    get_greedy_with_penalties(),
+    get_greedy_with_min_and_max_tokens(),
+    get_greedy_with_single_stop_string(),
+    get_greedy_with_multiple_stop_strings(),
+    get_greedy_with_multiple_stop_strings_no_match(),
+    get_greedy_stop_strings_exclude_from_output(),
+    get_greedy_stop_strings_include_to_output(),
+    get_greedy_n_stop_strings_exclude_from_output(),
+    get_greedy_n_stop_strings_include_to_output()
```

Comment:
do we really need all these variants? here you are rather testing Sampler than static pipeline.

Sampler itself is tested in test_sampling.py 

I suppose you can keep greedy and something with stop strings (to ensure that GenerationHandle is used instead of manual reading from SequenceGroup)